frontend/app/work_packages/controllers/work-packages-list-controller.js

```diff
@@ -115,7 +115,7 @@ module.exports = function($scope, $rootScope, $state, $location, latestTab,
     setupWorkPackagesTable(json);
 
     if (json.work_packages.length) {
-      $scope.preselectedWorkPackageId = json.work_packages[0].id;
+      WorkPackageService.cache().put('preselectedWorkPackageId', $scope.preselectedWorkPackageId);
```

Comment:
Copy & Paste error?

You are storing `$scope.preselectedWorkPackageId` in the cache... which looks really like the former cache variable that you are trying to eliminate here.

I guess you wanted to set `json.work_packages[0].id`